phc/easy/frame.py

```diff
@@ -159,14 +159,28 @@ def expand(
             local_key = f"{column_key}.local"
             tz_key = f"{column_key}.tz"
 
-            utc = pd.to_datetime(combined[column_key], utc=True)
-
-            # Cleverness: Use regex to remove TZ and parse as utc=True to
-            # produce local datetime. The column name will have ".local" as
-            # suffix so it'll be clear what's happening.
-            localized = pd.to_datetime(
-                combined[column_key].str.replace(TZ_REGEX, ""), utc=True
-            )
+            try:
+                utc = pd.to_datetime(combined[column_key], utc=True)
+
+                # Cleverness: Use regex to remove TZ and parse as utc=True to
+                # produce local datetime. The column name will have ".local" as
+                # suffix so it'll be clear what's happening.
+                localized = pd.to_datetime(
+                    combined[column_key].str.replace(TZ_REGEX, ""), utc=True
+                )
+            except pd.errors.OutOfBoundsDatetime as ex:
+                print(
```

Comment:
Since this repo is getting robust, you should start using the Python logging facility instead of print when possible.

Reply:
Sounds good. I'll add an issue to use it. 👍